packages/lab-extension/src/index.ts

```diff
@@ -90,22 +91,33 @@ const interfaceSwitcher: JupyterFrontEndPlugin<void> = {
     };
 
     const addInterface = (option: ISwitcherChoice) => {
-      const { command, commandLabel, urlPrefix } = option;
+      const { command, commandLabel, commandPaletteLabel, urlPrefix } = option;
+      const paletteCommand = `${command}-palette`;
+
+      const execute = () => {
+        const current = notebookTracker.currentWidget;
+        if (!current) {
+          return;
+        }
+        window.open(`${urlPrefix}${current.context.path}`);
+      };
+
       commands.addCommand(command, {
         label: (args) => (args.noLabel ? '' : commandLabel),
         caption: commandLabel,
-        execute: () => {
-          const current = notebookTracker.currentWidget;
-          if (!current) {
-            return;
-          }
-          window.open(`${urlPrefix}${current.context.path}`);
-        },
+        execute,
+        isEnabled,
+      });
+
+      commands.addCommand(paletteCommand, {
+        label: (args) => (args.noLabel ? '' : commandPaletteLabel),
+        caption: commandLabel,
+        execute,
         isEnabled,
       });
 
       if (palette) {
-        palette.addItem({ command, category: 'Other' });
+        palette.addItem({ command: paletteCommand, category: 'Other' });
```

Comment:
We should update the menu entries as well. For example by using the palette command here:

https://github.com/jupyter/notebook/blob/1ea8e1e3aa02dbad11ea31b9122d3b4e52d9cd26/packages/lab-extension/schema/interface-switcher.json#L11-L20

Reply:
updated naming conventions and menu entries, now they use "Open In [...]" strings